Controller/Component/WizardComponent.php

```diff
@@ -382,7 +382,7 @@ public function process($step) {
 					} else {
 						throw new NotImplementedException(sprintf(__('Process Callback not found. Please create Controller::%s', $processCallback)));
 					}
-					if ($proceed) {
+					if ($proceed === true) {
```

Comment:
$proceed from line 379 and line 381 should return always boolean, that's why it's a matter of best practices, to test type when dealing with booleans